### ISSUE TITLE
How to keep line breaks consistent

### ISSUE THREAD

**Issue by sullenor** [postcss/postcss#1021]
Hi, 

I have the following code and can't figure it out how to keep line breaks between different rules similar after `append` call. Selectors stick to the closing `}`, so they appear to be on the same line unlike in the begging. I hope you can help me here :)

```js
const postcss = require("postcss");

const ast = postcss.parse('/**/\n\n.a\n{}\n\n.b\n{}');
const root = postcss.root({raws: ast.raws});

root.append(ast.nodes);
console.log(root.toString()); // "/**/.a\n{}.b\n{}" <-- no \n\n between "}.b"
```

**Comment by ai** [comment]
Yeap, it is not cool. Maybe it is some side effect of `Root#normalize` hack?

I will look into in when I return from Cuba.

Right now you can do:

```js
root.nodes = ast.nodes.map(i => i.clone({ parent: root }))
```

**Comment by sullenor** [comment]
Yeah, I guess `Root#normalize` can be an issue, but not 💯 sure here (https://github.com/postcss/postcss/blob/master/lib/root.es6#L42-L44).

Thank you for the work around, I'll check it.

**Comment by ai** [comment]
Done 5744739

**Comment by ai** [comment]
Released in 6.0.2
